src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends Component {
+  render = () => (
+    <div>
+      <div className="container">
+        <span className="display">0</span>
+        <button type="submit">AC</button>
+        <button type="submit">+/-</button>
+        <button type="submit">%</button>
+        <button className="orange-cl" type="submit">
+          /
+        </button>
+        <button type="submit">7</button>
+        <button type="submit">8</button>
+        <button type="submit">9</button>
+        <button className="orange-cl" type="submit">
+          x
+        </button>
+        <button type="submit">4</button>
+        <button type="submit">5</button>
+        <button type="submit">6</button>
+        <button className="orange-cl" type="submit">
+          -
+        </button>
+        <button type="submit">1</button>
+        <button type="submit">2</button>
+        <button type="submit">3</button>
+        <button className="orange-cl" type="submit">
+          +
+        </button>
```

Comment:
Kindly, match the design given with the `+` and `/` buttons position. Also you can center the whole calculator in the page and give more padding inside the buttons and decrease the width of them to match the design.

| Your Design | Figma Design |
|-------------|---------------|
| <img width="1014" alt="image" src="https://user-images.githubusercontent.com/34578631/176429106-68e56069-30ac-4ed2-9646-933ddca00b28.png"> | <img width="538" alt="image" src="https://user-images.githubusercontent.com/34578631/176429236-5af6a543-93ca-4b03-aab6-268af390d85b.png"> |